_includes/all_keypoints.html

```diff
@@ -7,7 +7,7 @@ <h2>Key Points</h2>
   {% unless episode.break %}
     <tr>
       <td class="col-md-3">
-        <a href="{{ page.root }}{{ episode.url }}">{{ episode.title }}</a>
+        <a href="{{ base }}{{ episode.url }}">{{ episode.title }}</a>
```

Comment:
I'd suggest calling this variable something like `lesson_url`

Reply:
given that this variable stores repetitions of `../`, should we call it `relative_path` or `root_location`, or maybe `base_path`, I don't think that `lesson_url` is very accurate, and I agree that `base` is too vague.

Reply:
oh, right, I forgot this is a relative path... I like the `root_location` and `base_path` (with a slight preference for the former). Perhaps, we should add `relative` to make it clear that this is relative path for root directory?

`relative_root_path`, `relative_root_location`, ...?